engine/graphics/src/dx12/graphics_dx12.cpp

```diff
@@ -502,6 +526,45 @@ namespace dmGraphics
                 rtv_handle.Offset(1, context->m_RtvDescriptorSize);
             }
 
+            // Create the depth/stencil attachment(s)
+            D3D12_RESOURCE_DESC depth_desc = {};
+            depth_desc.Dimension           = D3D12_RESOURCE_DIMENSION_TEXTURE2D;
+            depth_desc.Width               = window_width;
+            depth_desc.Height              = window_height;
+            depth_desc.DepthOrArraySize    = 1;
+            depth_desc.MipLevels           = 1;
+            depth_desc.Format              = depth_format;
+            depth_desc.SampleDesc.Count    = context->m_MSAASampleCount;
+            depth_desc.SampleDesc.Quality  = 0;
+            depth_desc.Layout              = D3D12_TEXTURE_LAYOUT_UNKNOWN;
+            depth_desc.Flags               = D3D12_RESOURCE_FLAG_ALLOW_DEPTH_STENCIL;
+
+            D3D12_CLEAR_VALUE depth_clear    = {};
+            depth_clear.Format               = depth_format;
+            depth_clear.DepthStencil.Depth   = 1.0f;
+            depth_clear.DepthStencil.Stencil = 0;
+
+            hr = context->m_Device->CreateCommittedResource(
+                &CD3DX12_HEAP_PROPERTIES(D3D12_HEAP_TYPE_DEFAULT),
+                D3D12_HEAP_FLAG_NONE,
+                &depth_desc,
+                D3D12_RESOURCE_STATE_DEPTH_WRITE,
+                &depth_clear,
+                IID_PPV_ARGS(&context->m_FrameResources[i].m_DepthStencil)

```

Comment:
Try to use our `DM_IID_PPV_ARGS` we use on the xbox branch, as they're different on the platforms.
Makes it easier to merge.
(No, we haven't moved that from xbox branch to public branch yet afaik)